tests/loader/test_colmap_loader.py

```diff
@@ -0,0 +1,78 @@
+
+import unittest
+from pathlib import Path
+
+import numpy as np
+from gtsam import Pose3
+
+from gtsfm.common.image import Image
+from gtsfm.loader.colmap_loader import ColmapLoader
+
+TEST_DATA_ROOT = Path(__file__).resolve().parent.parent / "data"
+
+
+class TestColmapLoader(unittest.TestCase):
+
+    def setUp(self):
+        """Set up the loader for the test."""
+        super().setUp()
+
+        colmap_files_dirpath = TEST_DATA_ROOT / "set1_lund_door/colmap_ground_truth"
+        images_dir = TEST_DATA_ROOT / "set1_lund_door/images"
```

Comment:
Should we define these globally instead of the setup function?

Reply:
i was planning to add more unit tests in the future (perhaps with invalid paths and other directories) once we decide what we want the expected behavior to be. Maybe we can discuss that at our next meeting.

So I prefer not global for now.